app/controllers/spree/admin/payment_methods_controller_decorator.rb

```diff
@@ -10,7 +10,7 @@ module Admin
       # Only show payment methods that user has access to and sort by distributor name
       # ! Redundant code copied from Spree::Admin::ResourceController with modifications marked
       def collection
-        return parent.send(controller_name) if parent_data.present?
+        return parent.controller_name if parent_data.present?
```

Comment:
[controller_name](https://apidock.com/rails/v3.2.13/ActionController/Metal/controller_name/class) is a public method

Reply:
The original did not call `send(:controller_name)`. Doesn't it call a method with the same name as the controller?

Reply:
:hankey: yes, I went too fast on this one. Thanks for the heads-up!